src/FbgemmFP16.cc

```diff
@@ -208,7 +200,7 @@ FBGEMM_API void cblas_gemm_compute(
   const int n = Bp.numCols(), k = Bp.numRows(), ldc = n;
   const int mb_max = 120;
   constexpr int simd_width = 8;
-  constexpr int kernel_ncol_blocks = 1;
+  constexpr int kernel_ncol_blocks = 2;
```

Comment:
Maybe put this somewhere in FbgemmFP16.h and use it for when computing bcol_ ?

Reply:
Added "kernel_ncol_blocks" in FbgemmFP16.h and used it for computing bcol_.